app/main.js

```diff
@@ -20,9 +14,21 @@ import { setExceptionHandlers, checkPrevCrashes } from './crashes';
 // Show request in chrome network tool
 // GLOBAL.XMLHttpRequest = GLOBAL.originalXMLHttpRequest || GLOBAL.XMLHttpRequest;
 
+function setCodePush() {
+  const codepushEnable = false; // !__DEV__
```

Comment:
Dead kitten 🐱

Reply:
I thought it is useful until we have the first release. But is a good catch to remind it!